config/test.exs

```diff
@@ -22,6 +22,5 @@ config :changelog, Changelog.Repo,
   username: System.get_env("DB_USER", "postgres")
 
 config :changelog, Oban,
-  crontab: false,
```

Comment:
This hasn't been necessary for a while now. The top-level `crontab` option is deprecated, as CRON is implemented as a plugin now.